APIDesafioWarren/Controllers/CustomerController.cs

```diff
@@ -20,7 +21,14 @@ public CustomersController(ICustomerServices database)
         [HttpGet]
         public IActionResult Get()
         {
-            return Ok(_customerServices.GetAll());
+            return SafeAction(() =>
+            {
+               var findCustomers = _customerServices.GetAll();
+
+               return findCustomers.Count is 0 
+                   ? NotFound("Customer not found!")
+                   : Ok(findCustomers);
```

Comment:
Não estamos fazendo nenhum filtro aqui para buscar dados, caso não encontre apenas significa que não existe nenhum dado no banco e não que não foi encontrado

```suggestion
               return findCustomers.Count is 0 
                   ? Ok()
                   : Ok(findCustomers);
```